xknx/devices/climate.py

```diff
@@ -68,6 +69,7 @@ def __init__(
         group_address_fan_speed: GroupAddressesType = None,
         group_address_fan_speed_state: GroupAddressesType = None,
         fan_speed_mode: FanSpeedMode = FanSpeedMode.PERCENT,
+        group_address_humidity: GroupAddressesType = None,
```

Comment:
```suggestion
        group_address_humidity_state: GroupAddressesType = None,
```
Since this is a state address, it should follow the naming schema.